.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,21 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.9"
+
+      - name: Pre-commit
+        uses: pre-commit/action@v3.0.1
```

Comment:
From what I can gather this uses the standard template for pre-commit which is good. How do we maintain this?

Reply:
we could set up dependabot here, similar to in the msig repo: https://github.com/BalancerMaxis/multisig-ops/blob/main/.github/dependabot.yaml